### PR TITLE
Carousel Block (#101)

### PR DESCRIPTION
**Pull request by arumsey**
Fix #101

Added a carousel block for the testimonials section of the website. Carousel block has feature parity with existing carousel but has zero third party dependencies.

- smotth scrolling
- drag support
- auto height calculations
- navigation buttons
- auto navigation
- page and scroll visibility aware
- show more link
- accessibility

Test URLs:
Before: https://main--merative2--hlxsites.hlx.page/drafts/arumsey/clinical-development
After: https://carousel--merative2--hlxsites.hlx.page/drafts/arumsey/clinical-development

image

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1 +1,3 @@
-helix-importer-ui
\ No newline at end of file
+helix-importer-ui
+scripts/jquery*.js
```
Comment (arumsey) [inline .eslintignore:2]: I need to remove these

File: blocks/carousel/carousel.css [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    -webkit-background-clip: text;     /* stylelint-disable-line */
+    -webkit-text-fill-color: transparent;
+    color: transparent;
+    border-image-slice: 1;
+    border-image-source: var(--gradient-left-right);
+}
+
+/* Media queries */
+
+@media screen and (min-width: 1024px) {
```
Comment (amol-anand) [inline blocks/carousel/carousel.css:157]: This should be min-width: 1200px if you're targeting desktop

File: blocks/carousel/carousel.css [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+.carousel .carousel-nav {
+    opacity: 0;
+    z-index: 2;
+    top: 125px;
+    height: 54px;
+    width: 54px;
+    position: absolute;
+    transition: .5s;
+    cursor: pointer;
+    background: url('../../icons/testimonial.svg');
```
Comment (amol-anand) [inline blocks/carousel/carousel.css:77]: We have the standard purple arrow already in the icons folder. You could reuse the same. There's also a gradient in the global vars you can use for hover effect for the colorful hover state

**Review by amol-anand — CHANGES_REQUESTED** [review]

File: blocks/carousel/carousel.css (hunk 2 above)
Comment (arumsey) [reply blocks/carousel/carousel.css]: Fixed

**Review by amol-anand — APPROVED** [review]

**Pull request merged**